src/position.ts

```diff
@@ -847,6 +815,20 @@ export class UserPosition extends PositionWithRunner {
     return createEmptyPermitSignature();
   }
 
+  private async sendManagePositionTransaction<A extends Array<unknown>, R, S extends Exclude<StateMutability, 'view'>>(
+    method: TypedContractMethod<A, R, S>,
+    gasLimitMultiplier: Decimal,
+    args: { [I in keyof A]-?: A[I] },
+  ): Promise<ContractTransactionResponse> {
+    if (!gasLimitMultiplier.equals(Decimal.ONE)) {
+      const gasEstimate = await method.estimateGas(...args);
+      const gasLimit = new Decimal(gasEstimate, Decimal.PRECISION).mul(gasLimitMultiplier).toBigInt();
+      return await method(...args, { gasLimit } as Overrides<S>);
```

Comment:
We need to pass `value` here in case ETH is used for collateral `positionManagerStEth.managePositionETH()`.

Reply:
Right, good catch